lib/instagram.js

```diff
@@ -41,7 +41,7 @@ var instagram = function(spec, my) {
   my.limit = null;
   my.remaining = null;
   my.agent = spec.agent;
-  my.host = spec.host || 'api.instagram.com';
+  my.host = spec.host || 'https://api.instagram.com';
```

Comment:
I'd prefer to use a `my.port` to specify another port, so that everything is clear. As `my.host` name defines it, this is just the host, not the full URL.